config/Config-images.in

```diff
@@ -153,6 +153,38 @@ menu "Target Images"
 		help
 		  Build a squashfs-lzma root filesystem.
 
+		config TARGET_SQUASHFS_COMPRESSION_OVERRIDE
+			bool "Override compression"
+			default n
+			depends on TARGET_ROOTFS_SQUASHFS
+			help
+			  Override default compression method
+
+		choice
+			prompt "Compression"
+			default TARGET_SQUASHFS_COMPRESSION_GZIP
+			depends on TARGET_ROOTFS_SQUASHFS
+			depends on TARGET_SQUASHFS_COMPRESSION_OVERRIDE
+			help
+			  Select squashfs compression.
+
+			config TARGET_SQUASHFS_COMPRESSION_GZIP
+				bool "gzip"
+
+			config TARGET_SQUASHFS_COMPRESSION_LZ4
+				bool "lz4"
+
+			config TARGET_SQUASHFS_COMPRESSION_LZO
+				bool "lzo"
+
+			config TARGET_SQUASHFS_COMPRESSION_XZ
+				bool "xz"
+
+			config TARGET_SQUASHFS_COMPRESSION_ZSTD
+				depends on !LINUX_5_4 && !LINUX_4_19
```

Comment:
OpenWrt master only supports kernel 5.10 and later. these dependencies are not needed.

Reply:
Copied from the target package, which also has these, they should probably be nitpicked over there too?